### ISSUE TITLE
Truffle is being sunset 

### ISSUE THREAD

**Issue by MadelineMurray** [ConsenSys-archive/truffle#6204]
Pinning this issue for visibility 

⚠️  The Truffle Suite is being sunset. For information on ongoing support, migration options and FAQs, visit the [Consensys blog](https://consensys.io/blog/consensys-announces-the-sunset-of-truffle-and-ganache-and-new-hardhat?utm_source=github&utm_medium=referral&utm_campaign=2023_Sep_truffle-sunset-2023_announcement_). Thank you for all the support over the years.

**Comment by mr-possible** [comment]
But does that mean after 90 days Truffle and Ganache won't be available to use or what? I didn't quite understand the word 'sunset'. Can someone please explain to me in layman's terms?

**Comment by MadelineMurray** [comment]
@mr-possible - they'll remain available but this repository will be moved to an archived state and no further updates or development will happen.

**Comment by alexsupa597** [comment]
> @mr-possible - they'll remain available but this repository will be moved to an archived state and no further updates or development will happen.

Are there any reasons?
